tests/integration/components/modals/paytm-otp-test.js

```diff
@@ -0,0 +1,19 @@
+import { module, test } from 'qunit';
+import { setupIntegrationTest } from 'open-event-frontend/tests/helpers/setup-integration-test';
+import hbs from 'htmlbars-inline-precompile';
+import { render } from '@ember/test-helpers';
+
+module('Integration | Component | modals/paytm otp modal', function(hooks) {
+  setupIntegrationTest(hooks);
+
+  test('it renders', async function(assert) {
+    this.setProperties({
+      'isOpen'    : false,
+      'currency'  : 'USD',
+      'amount'    : 100,
+      'verifyOtp' : () => {}
```

Comment:
This looks fine but what does this test exactly?

Reply:
In the current state, it checks rendering of the modal without actually hitting the API (gotta save the API calls xD  )